tests/e2e/provisioning/test/e2e_provisioning_test.go

```diff
@@ -46,4 +49,33 @@ func Test_E2E_Provisioning(t *testing.T) {
 
 	err = ts.dashboardChecker.AssertRedirectedToUAA(dashboardURL)
 	assert.NoError(t, err)
+
+	if ts.IsTestAzureEventHubsEnabled {
+		checkAzureEventHubProperties(ts, t, operationID)
+	}
+}
+
+func checkAzureEventHubProperties(ts *Suite, t *testing.T, operationID string) {
+	resourceGroup, err := (*ts.azureClient).GetResourceGroup(context.TODO(), ts.brokerClient.GetClusterName())
+	assert.NoError(t, err)
+	assert.Equal(t, http.StatusOK, resourceGroup.StatusCode, "HTTP GET fails for ResourceGroup")
+	assert.NotNil(t, resourceGroup, "ResourceGroup name is nil")
+	assert.Equal(t, ts.brokerClient.GetClusterName(), *resourceGroup.Name, "ResourceGroup Name is incorrect")
+
+	// Check tags for ResourceGroup
+	assert.Equal(t, operationID, *resourceGroup.Tags[azure.TagOperationID], "Tag OperationID for ResourceGroup is incorrect")
+	assert.Equal(t, ts.brokerClient.InstanceID(), *resourceGroup.Tags[azure.TagInstanceID], "Tag InstanceID for ResourceGroup is incorrect")
+	assert.Equal(t, ts.brokerClient.SubAccountID(), *resourceGroup.Tags[azure.TagSubAccountID], "Tag SubAccountID for ResourceGroup is incorrect")
+
+	namespace, err := (*ts.azureClient).GetEHNamespace(context.TODO(), ts.brokerClient.GetClusterName(), ts.brokerClient.GetClusterName())
+	assert.NoError(t, err)
+	assert.Equal(t, http.StatusOK, resourceGroup.StatusCode, "HTTP GET fails for EH Namespace")
+	assert.NotNil(t, namespace, "EH Namespace name is nil")
```

Comment:
Your not checking the namespaces name here

Reply:
require would be the better option, as later asserts will try to dereference nil otherwise

Reply:
Again, checking the name of the NS is not critical here.